cmd/main.go

```diff
@@ -28,11 +29,15 @@ func main() {
 	c, err := cli.New(
 		cli.WithCommandName("kubebuilder"),
 		cli.WithVersion(versionString()),
+		cli.WithDefaultProjectVersion(config.Version2),
 		cli.WithPlugins(
 			&pluginv2.Plugin{},
 			&pluginv3.Plugin{},
 		),
-		cli.WithDefaultPlugins(
+		cli.WithDefaultPlugins(config.Version2,
+			&pluginv2.Plugin{},
+		),
+		cli.WithDefaultPlugins(config.Version3Alpha,
 			&pluginv2.Plugin{},
```

Comment:
how we can have 2 default plugins?

Reply:
Default plugin for different project versions.
`kubebuilder init --project-version 2` will use the first set and `kubebuilder init --project-version 3-alpha` will use the second set.